llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -410,6 +408,9 @@ struct VPTransformState {
 
   /// VPlan-based type analysis.
   VPTypeAnalysis TypeAnalysis;
+
+  /// Updater for the DominatorTree.
+  DomTreeUpdater DTU;
```

Comment:
nit: place DTU in DT's position (after LI), or inside CFG?

Reply:
Now inside CFG